src/app/(project)/projects/[project]/drops/Drops.tsx

```diff
@@ -294,31 +280,59 @@ export default function Drops({ project }: DropsPageProps) {
-                                  ),
-                                }),
+                                // columnHelper.accessor('endTime', {
+                                //   header: () => <span>Mint end date</span>,
+                                //   cell: (info) => (
+                                //     <div className="flex flex-col gap-1">
+                                //       {info.getValue() ? (
+                                //         <>
+                                //           <span className="text-xs text-gray-400 font-medium">
+                                //             {format(
+                                //               convertLocalTime(info.getValue()),
+                                //               DateFormat.DATE_1
+                                //             )}
+                                //           </span>
+                                //           <span className="text-xs text-white">
+                                //             {format(
+                                //               convertLocalTime(info.getValue()),
+                                //               DateFormat.TIME_1
+                                //             )}
+                                //           </span>
+                                //         </>
+                                //       ) : (
+                                //         <span className="text-xs text-white font-medium">None</span>
+                                //       )}
+                                //     </div>
+                                //   ),
+                                // }),
+                                columnHelper.accessor(
+                                  ({ collection, status }) => {
+                                    if (!collection) {
+                                      throw new Error('no collection');
+                                    }
+
+                                    return {
+                                      supply: collection?.supply,
+                                      totalMints: collection?.totalMints,
+                                      status,
+                                    };
+                                  },
+                                  {
+                                    id: 'supply',
+                                    header: () => <span>Supply</span>,
+                                    cell: (info) => {
+                                      const { supply, totalMints, status } = info.getValue();
+
+                                      return (
+                                        <div className="flex items-center text-white text-xs font-medium">
+                                          {`${supply || 0}${totalMints ? ' / ' : ''}${
+                                            totalMints || ''
+                                          } minted`}
+                                        </div>
```

Comment:
If the supply is null then its unlimited mints so we shouldn't show the progress bar and write the supply as `Unlimited`

Reply:
Added a simple infinity symbol for now. Will update it once I get the design.